src/streams/StreamList.js

```diff
@@ -75,6 +75,11 @@ export default class StreamList extends PureComponent<Props> {
             isSwitchedOn={item.subscribed}
             onPress={onPress}
             onSwitch={onSwitch}
+            accessibilityLabel={`${item.name} ${
```

Comment:
This text can be translated too, even though it has special data values (the topic's name and unread count) in it. See, e.g., `alertTitle` in src/compose/MentionWarnings.js:

```js
    const alertTitle = _('Couldn’t load information about {fullName}', {
      fullName: mentionedUser.full_name,
    });
```